core/src/main/scala/com/microsoft/azure/synapse/ml/causal/DoubleMLParams.scala

```diff
@@ -85,6 +117,22 @@ trait DoubleMLParams extends Params
    */
   def setSampleSplitRatio(value: Array[Double]): this.type = set(sampleSplitRatio, value)
 
+  object TreatmentTypes extends Enumeration {
+    type TreatmentType = Value
+    val Binary, Continuous = Value
+  }
+
+  def getTreatmentType: TreatmentTypes.Value = {
+    val treatmentType =
+      getTreatmentModel match {
+        case _: ProbabilisticClassifier[_, _, _] =>
+          TreatmentTypes.Binary
+        case _: Regressor[_, _, _] =>
+          TreatmentTypes.Continuous
+      }
+    treatmentType
+  }
```

Comment:
Should we expose these two definitions as public? Or set it to private package accessibility?

Reply:
will set them as private